src/helpers/git_helpers/open_changes_helpers.sh

```diff
@@ -40,12 +40,24 @@ function restore_open_changes {
 }
 
 
+# Unstashes changes silently (no console output)
+function restore_open_changes_silently {
```

Comment:
Same here, make "silently" a parameter to the normal `restore_open_changes` method.